### PR TITLE
[IMP] account_invoice_overdue_reminder: Add "Other" Reminder Type

### PR DESCRIPTION
**Pull request by SirTakobi**
Supersede https://github.com/OCA/credit-control/pull/202.

### CODE REVIEW

File: account_invoice_overdue_reminder/models/overdue_reminder_action.py
Comment (legalsylvain) [inline account_invoice_overdue_reminder/models/overdue_reminder_action.py]: Other seems to generic for me. Something like 'manual' looks better. Don't you think ? 

same for the function, so validate_other --> validate_manual.

**Review by legalsylvain — COMMENTED** [review]
One remarks. Otherwise, LGTM. 
Thanks !

File: account_invoice_overdue_reminder/models/overdue_reminder_action.py
Comment (SirTakobi) [reply account_invoice_overdue_reminder/models/overdue_reminder_action.py]: Thanks for the review!
I have changed the code as you suggested, can you have another look?

**Review by legalsylvain — APPROVED** [review]
Thanks !

**Comment by alexis-via** [comment]
Just for my understanding: what kind of real-life scenario do you put under this new type manual/other ?

**Comment by SirTakobi** [comment]
> Just for my understanding: what kind of real-life scenario do you put under this new type manual/other ?

Asked this to the functional guys, they say that this can be used when the salesman goes see the customer, or the salesman leaves a message to the customer.
cc @eLBati

**Review by mrcast — APPROVED** [review]
Functional test, ok!

**Comment by SirTakobi** [comment]
@OCA/accounting-maintainers this has been approved after the bot marked it as stale, can someone reopen it? I can't :(
Thanks!

**Pull request closed without merging**
